lib/dynflow/web/navbar_items_helper.rb

```diff
@@ -0,0 +1,17 @@
+module Dynflow
+  module Web
+    module NavbarItemsHelper
+      module Link
+        @@link_back ||= nil
```

Comment:
`lhs ||= rhs` sets `lhs` to the value of `rhs` if `lhs` is `nil`. Setting it to `nil` if it is `nil` doesn't make much sense